molgenis-data-mapper/src/main/resources/templates/view-attribute-mapping.ftl

```diff
@@ -34,7 +34,7 @@
 						</tr>
 					</thead>
 					<tbody>
-						<#list entityMapping.sourceEntityMetaData.attributes as source>
+						<#list suggestedAttributes as source>
 							<tr>
```

Comment:
Maybe if one list is empty write down the other? Or allow the user to switch between the two?